run/jenkins.py

```diff
@@ -0,0 +1,282 @@
+# This replaces run/run.py
+import json
+import gzip
+import os
+import requests
+import subprocess
+import time
+import re
+
+
+METADATA_URL = 'http://metadata.google.internal/computeMetadata/v1'
+
+
+def main():
+    # TODO after --install-browser verify browser
```

Comment:
With @mattl, @mdittmer, @lukebjerring, @jeffcarp and possibly others now on this project, should we start assigning owners to TODOs?

Reply:
(just some context on this file specifically - `run/jenkins.py` in this PR is a clone of `run/run.py` adapted to run in the container. Once this lands and the container supports all platforms, we can delete `run/run.py`)

Reply:
To answer your question: I can make bugs for all of these and we can decide on owners later.

Reply:
👍

Reply:
https://github.com/w3c/wptdashboard/issues/203
https://github.com/w3c/wptdashboard/issues/204